netebpfext/net_ebpf_ext_hook_provider.c

```diff
@@ -4,7 +4,11 @@
 #include "ebpf_extension_uuids.h"
 #include "net_ebpf_ext_hook_provider.h"
 
-#define NET_EBPF_EXT_STACK_EXPANSION_SIZE 1024 * 10
+#ifdef _DEBUG
+#define NET_EBPF_EXT_STACK_EXPANSION_SIZE 1024 * 20
```

Comment:
The latest crash dump shows the stack is about 23K bytes. With the frames above the call to the ETW API around 9K. I suggest giviing it one more try with increasing the stack by 32K for debug.